src/context/data-context.js

```diff
@@ -5,6 +5,7 @@ const DataContext = createContext();
 
 const DataProvider = ({ children }) => {
   const [loading, setLoading] = useState(false);
+  const [questionCount, setQuestionCount] = useState(1);
   const [isOpen, setIsOpen] = useState(true);
```

Comment:
give a meaningful name for isOpen, its not declarative for what it is being used for